src/wazuh_modules/vulnerability_scanner/tests/CMakeLists.txt

```diff
@@ -6,6 +6,10 @@ include_directories(${CMAKE_SOURCE_DIR})
 include_directories(${CMAKE_SOURCE_DIR}/include/)
 include_directories(${SRC_FOLDER}/external/googletest/googletest/include/)
 include_directories(${SRC_FOLDER}/external/googletest/googlemock/include/)
+include_directories(${SRC_FOLDER}/wazuh_modules/vulnerability_scanner/src/)
+
 link_directories(${SRC_FOLDER}/external/googletest/lib/)
 
 add_subdirectory(policyManager)
+add_subdirectory(unit)
+add_subdirectory(component)
```

Comment:
Improve: Use the UNIT_TEST=ON "flag" to prevent tests from compiling when is not required

Reply:
it's not already handled by [this](https://github.com/wazuh/wazuh/blob/98d1f9ffc20195c8feef247998ccebedf12c6eea/src/wazuh_modules/vulnerability_scanner/CMakeLists.txt#L56)?

Reply:
It is!